modules/vaos/app/services/vaos/middleware/response/errors.rb

```diff
@@ -12,38 +12,38 @@ def on_complete(env)
           Raven.extra_context(vamf_status: env.status, vamf_body: env.body, vamf_url: env.url)
           case env.status
           when 400, 409
-            error_400(env.body)
```

Comment:
@grossb1 now that my PR has been merged, we can probably try to handle the 409 status code using exception.en.yml

The changes we're waiting on from VAOS BE haven't hit production yet, but I think we could still make the change so that it works as expected when those changes go live.

Reply:
I included the condition for 409s